x-pack/plugins/enterprise_search/public/applications/workplace_search/views/content_sources/components/add_source/add_source_logic.ts

```diff
@@ -430,9 +419,7 @@ export const AddSourceLogic = kea<MakeLogicType<AddSourceValues, AddSourceAction
         ? `/internal/workplace_search/org/sources/${sourceId}/reauth_prepare`
         : `/internal/workplace_search/account/sources/${sourceId}/reauth_prepare`;
 
-      const query = {
-        kibana_host: kibanaHost,
-      } as HttpFetchQuery;
+      const query = {} as HttpFetchQuery;
```

Comment:
Is there a reason to send `query` at all? My [comment below](https://github.com/elastic/kibana/pull/119974/files#r759387727) still stands. I'm surprised that this works, as the kibana API proxy has it removed below, so it should error out in the browser.